### ISSUE TITLE
Syndicate banhammer issue

### ISSUE THREAD

**Issue by Maroonenahmoron** [HippieStation/HippieStationdeprecated2020#4408]
It's pretty simple; it does no damage on melee despite its force being 20 but it does deal the 20 throwforce and makes a **TWHACK** when tossed at someone.

The banhammer needs to remain stealthy (despite its red tagged appearance) and still be able to do its proper damage.

**Comment by Arzion5** [comment]
I'll look into it, thanks for reporting the issue.

**Comment by Pyko1** [comment]
```
/obj/item/banhammer/attack(mob/M, mob/user)
	if(user.zone_selected == "head")
		M.visible_message("<span class='danger'>[user] are stroking the head of [M] with a bangammer</span>", "<span class='userdanger'>[user] are stroking the head with a bangammer</span>", "you hear a bangammer stroking a head");

	else
		M.visible_message("<span class='danger'>[M] has been banned FOR NO REISIN by [user]</span>", "<span class='userdanger'>You have been banned FOR NO REISIN by [user]</span>", "you hear a banhammer banning someone")
	playsound(loc, 'sound/effects/adminhelp.ogg', 15) //keep it at 15% volume so people don't jump out of their skin too much
```

This is overwriting the normal attack code.

**Comment by KayeArray** [comment]
lol

**Comment by JohnGinnane** [comment]
Fixed by #4413
